src/clipboard.rei

```diff
@@ -0,0 +1,5 @@
+[@bs.module "react-native"] [@bs.scope "Clipboard"]
```

Comment:
Our convention is to use lets in the `rei`s. `external` is just an implementation detail